src/rust/grapl-graphql-codegen/src/node_type.rs

```diff
@@ -409,9 +409,9 @@ impl<'a> TryFrom<&ObjectType<'a, &'a str>> for NodeType {
     }
 }
 
-fn get_extends_type_name<'a>(
-    directives: &[Directive<'a, &'a str>],
-) -> Result<String, CodeGenError<'a>> {
+fn get_extends_type_name(
```

Comment:
This can be removed, as can all code related to "extending" a node/ schema.

Reply:
After experimenting with it a bit, I don't feel comfortable enough with this codebase to accurately rip these out.